### PR TITLE
po/fix cat support

### PR DESCRIPTION
**Pull request by TurboGit**

### CODE REVIEW

File: src/develop/imageop.c [hunk 1]
```diff
@@ -1204,10 +1204,8 @@ static void dt_iop_gui_off_callback(GtkToggleButton *togglebutton, gpointer user
     {
       module->enabled = 0;
 
-      //if current module is set as the CAT instance, remove that setting
-      dt_iop_order_entry_t *CAT_instance = module->dev->proxy.chroma_adaptation;
-
-      if(CAT_instance != NULL && CAT_instance->o.iop_order == module->iop_order)
+      //  if current module is set as the CAT instance, remove that setting
+      if(module->dev->proxy.chroma_adaptation == module)
```
Comment (aurelienpierre) [inline src/develop/imageop.c:1208]: I completely forgot about that part of the code

**Review by aurelienpierre — APPROVED** [review]
Looks good.

**Comment by kofa73** [comment]
Testing locally

**Comment by johnny-bit** [comment]
With this PR my replication steps do not replicate issue.

So looks good, just need to make sure there are no false-positives and false negatives, like @Mark-64 mentioned in #7271

**Comment by Mark-64** [comment]
Tested the PR against false negatives.
If I "duplicate instance" I get the warning, but if I do "new instance" and manually select a CAT there is no warning.
Is this the intended behaviour ?

**Comment by kofa73** [comment]
@Mark-64 : If I create a new instance, and select an adaptation, but don't modify the illuminant (leave it as 'same as pipeline (D50)'), there's no warning. If I switch it to another illuminant, the warning appears. I think that is the intended behaviour -- is it not?
image
vs
image

**Comment by Mark-64** [comment]
> @Mark-64 : If I create a new instance, and select an adaptation, but don't modify the illuminant (leave it as 'same as pipeline (D50)'), there's no warning. If I switch it to another illuminant, the warning appears. I think that is the intended behaviour -- is it not?

Same behaviour, if this is wanted then we can close the issue

**Pull request merged**